packages/blockly/src/components/maybe-file-modal.scss

```diff
@@ -0,0 +1,26 @@
+.modalForm {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  .modalInputGroup {
+    display: flex;
+    flex-direction: row;
+    gap: 5px;
+    align-items: center;
+
+    label {
+      display: inline;
+      white-space: pre-wrap;
+      font-weight: normal;
+    }
```

Comment:
The labels aren't quite vertically aligned with the input elements because the labels have a 5px bottom margin. If you add `margin-bottom: 0` here, it should fix that.

Reply:
Fixed in https://github.com/concord-consortium/question-interactives/pull/449/commits/0b2dac7964cfa2e0c0efe1d8613f03e39692b9bb.